Refresh button

christopher-dabrowski opened the pull request:
Added button for manual weather refresh

-------------------
⚠️  This branch is based of _api_ branch and should **not be merged** before it!  
After merge this branch can be safely **deleted** 💣

christopher-dabrowski committed:
Downloading weather

christopher-dabrowski committed:
Ignore vsc config

christopher-dabrowski committed:
Parse wather data

christopher-dabrowski committed:
Moc rain data

christopher-dabrowski committed:
Render current temperature

christopher-dabrowski committed:
Render rain percentage

christopher-dabrowski committed:
Add refresh button to website

christopher-dabrowski committed:
Make refresh button work

KrzeszewskiAdrian commented on index.css:
```diff
@@ -0,0 +1,20 @@
+
+.forecast-header .refresh-button {
+    background-color: #009ad8;
+    color: white;
```
Tu będzie problem z trybem dziennym i nocnym: zróbmy go w kolorze przycisku "find" na aktualnym mocku, bo jest pasujący dla dzień i noc (tj. #009ad8)

Nawet lepiej możnaby go zdefiniować jako button, żeby z css był zciągnięty wzór, ale to raczej wszystko jedno

christopher-dabrowski committed:
Refacotr - Save rain data to view model

Render function used to call mockRainData directry. To remove this dependency getWeatherData() now uses mock to ad rain data to model.
